pkg/okteto/pipeline.go

```diff
@@ -225,104 +307,187 @@ func areSameRepository(repoA, repoB string) bool {
 }
 
 // DestroyPipeline destroys a pipeline
-func DestroyPipeline(ctx context.Context, name, namespace string, destroyVolumes bool) (*GitDeployResponse, error) {
+func (c *OktetoClient) DestroyPipeline(ctx context.Context, name, namespace string, destroyVolumes bool) (*GitDeployResponse, error) {
 	log.Infof("destroy pipeline: %s/%s", namespace, name)
-	q := ""
+	gitDeployResponse := &GitDeployResponse{}
 	if destroyVolumes {
-		q = fmt.Sprintf(`mutation{
-			destroyGitRepository(name: "%s", space: "%s", destroyVolumes: %t){
-				action {
-					id,name,status
-				},
-				gitDeploy {
-					id,status
+		var mutation struct {
+			GitDeployResponse struct {
```

Comment:
Sounds a bit weird `GitDeployResponse` for the destroy operation, I guess it is because the function return an already existent `GitDeployResponse` type

Reply:
yes, that's the type the api is returning